modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java

```diff
@@ -152,12 +153,11 @@ public Map<String, PropertyDefinition> getExtendedPropertyDefinitions(
 				_objectRelationshipLocalService.getAllObjectRelationships(
 					objectDefinition.getObjectDefinitionId())) {
 
-			if (!Objects.equals(
-					objectRelationship.getType(),
-					ObjectRelationshipConstants.TYPE_MANY_TO_MANY) &&
-				!Objects.equals(
-					objectRelationship.getType(),
-					ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
+			Set<String> defaultObjectRelationshipTypes =
+				ObjectRelationshipUtil.getDefaultObjectRelationshipTypes();
+
+			if (!defaultObjectRelationshipTypes.contains(
+					objectRelationship.getType())) {
```

Comment:
Since we have many scenarios involving a condition, and we have to declare a variable + call the method contains, I'm wondering if it should be better to add a method that returns a boolean and receives the relationshipType as parameter :thinking: 
I'm bothered about creating this variable to only use in the IF condition.
What do you think?

Reply:
I agree. Should the method be call `isAllowedObjectRelationshipType(String type)`?